recipients_app/lib/data/repositories/user_repository.dart

```diff
@@ -59,18 +59,21 @@ class UserRepository {
 
   Future<void> verifyPhoneNumber({
     required String phoneNumber,
-    required Function(String) onCodeSend,
+    required Function(String, int?) onCodeSend,
     required Function(FirebaseAuthException) onVerificationFailed,
     required Function(PhoneAuthCredential) onVerificationCompleted,
+    required int? forceResendingToken,
   }) async {
     await firebaseAuth.verifyPhoneNumber(
       phoneNumber: phoneNumber,
+      forceResendingToken: forceResendingToken,
```

Comment:
Added this to make sure that the same phone can be provided twice even if `auto retrieval timeout` hasn't passed yet.